docs/Catalog/VisualizationDashboards.md

```diff
@@ -281,12 +281,29 @@ scipy
 Quilt renders tabular data formats into a Perspective Datagrid, including the
 following file extensions: .csv, .xls, .xlsx, .jsonl, .parquet, and .tsv. 
 
-For speed, Quilt loads a small preview of the rows stored in S3. You can click
-Load More to fetch up to about 6MB of zipped data. Beyond this size, click Download
-to see the entire file contents.
+For speed, Quilt loads the first few rows stored in S3. Click Load More to fetch
+up to about 6MB of zipped data. To see the entire file contents for large files,
+download the file (lower left).
 
-In order to open the analysis and visualization capabilities of Perspecitve,
-click the vertical ellipsis, upper left. To open the controls by default, set the `config.settings` property
+### Filter, plot, pivot
+
+Click Filter and Plot to open the side drawer. Drag and drop columns to Group By,
```

Comment:
"Drag and drop column names from the sidebar", maybe? It's not entirely clear that this is not about table columns